### ISSUE TITLE
Webgrind is not showing the analysis of the cachegrind file.

### ISSUE THREAD

**Issue by kmgalanakis** [Varying-Vagrant-Vagrants/VVV#2012]
With the latest `develop`, I'm using `/?XDEBUG_PROFILE` to build the cachegrind file of a page load, the cachegrind file is available on the webgrind's dropdown but upon hitting `Update`, even though webgrind seems to be loading something, the analysis never happens.

Am I doing something wrong?

**Comment by Mte90** [comment]
Do you have any errors?

**Comment by kmgalanakis** [comment]
Not any noticeable ones. Where do you want me to look?

**Comment by Mte90** [comment]
Webgrind is not so much developed so maybe is a bug of webgrind or the cachegrind file is not working. For the second one you can test with another viewer but if it is a webgrind bug we cannot do so much :-/

There is this as example https://github.com/jokkedk/webgrind/issues/5

**Comment by kmgalanakis** [comment]
With an external Viewer, I can properly inspect the generated files. Let's keep track of the webgrind tickets then. Maybe you should switch back to an older version of webgrind, if one was working before.

**Comment by tomjn** [comment]
Is your external viewer in the host or the guest? I’ll be honest webgrind
was here before I got into the project so I’ve never used it before

**Comment by kmgalanakis** [comment]
It seems that webgrind was actually producing an error that was only visible in the generic PHP error log.

Debugging the issue I found out that maybe I could resolve it. Trying to push a PR in the webgrind codebase, I found this https://github.com/jokkedk/webgrind/pull/121, which actually is the same as the one I was trying to push.

Hopefully, it will be merged relatively soon and will be available in the next release.
